LifeCycleAssessment_oM/LifeCycleAssessment_oM.csproj

```diff
@@ -34,6 +34,10 @@
       <HintPath>..\..\BHoM\Build\BHoM.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Geometry_oM, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Reference should not include `Version`, `Culture`, or `processorArchitecture`. Reference should also be set to `copy local` false